modules/simphera_aws_instance/postgresql.tf

```diff
@@ -1,3 +1,17 @@
+data "aws_subnet" "example" {

```

Comment:
This example should be changed with something meaningful like vpc_private_subnets, private_subnets or something like that and then reuse the same name in line 8.

Reply:
Done